src/app/common/edit-profile-form/edit-profile-form.component.ts

```diff
@@ -62,16 +62,11 @@ export class EditProfileFormComponent implements OnInit {
           this.user = updatedUser;
           this.initialFirstName = this.user.firstName;
 
-          // TODO: refactor into new alertService
-          // this is a new snackbar alert test
-          this._snackBar.open('Profile saved', 'dismiss', {
-            duration: 1500,
-            horizontalPosition: 'end',
-            verticalPosition: 'top',
-          });
         }
       },
-      error: (error) => console.log(error),
+      error: (error) => {
+        this.alertService.add(AlertType.DANGER, error);
```

Comment:
So this would become: `this.alertservice.danger(error)`

Reply:
commit: https://github.com/doubtfire-lms/doubtfire-web/pull/650/commits/62ac3e891f326d92b52d46bbc9b6208a2e96dbec